test/pkg/ptphelper/ptphelper.go

```diff
@@ -435,8 +435,9 @@ func WaitForPtpDaemonToBeReady(podList []*v1core.Pod) int {
 		if err != nil {
 			isOCP = false
 		}
-
-		if isOCP && semver.Compare(ocpVersion, "4.19") < 0 {
+		ocpVer, _ := semver.NewVersion(ocpVersion)
```

Comment:
I think this should actually be PTP operator version instead of OCP version as we don't redeploy the cluster each time now.